src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/AppInsights/AppInsightsLogger.cs

```diff
@@ -0,0 +1,312 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Web;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal class AppInsightsLogger : ILogger
```

Comment:
We try to keep the full name for the application insights - related classes (mostly because of marketing and there are other things called AppInsights)